pytim/gitim.py

```diff
@@ -199,24 +202,19 @@ def alpha_shape(self, alpha, group, layer):
             extraids = np.append(extraids, [-1] * n_cube)
         else:
             n_cube = 0
-            extrapoints = np.copy(points)
+            extrapoints = points
```

Comment:
without the copy, any manipulation on extrapoint will be reflected on points. This is not what we want.

Reply:
The only thing that we perform on extrapoints is the Delaunay() triangulation. Does it alter its input?

Reply:
No, but you don't want to alter points by adding to them the extra ones

Reply:
but that's it! We are not adding anything extra. If self._noextrapoints == True, then extrapoints simply coincides with points.